src/CashuWallet.ts

```diff
@@ -737,6 +771,10 @@ class CashuWallet {
 				options.privkey
 			).map((p: NUT11Proof) => serializeProof(p));
 		}
+		// Strip DLEQs if any
+		proofsToSend = proofsToSend.map((p: Proof) => {
```

Comment:
Do dleqs need to be removed or are you optimising for smaller payloads here?

Reply:
They need to be removed, otherwise you destroy your anonimity.

Reply:
Oh duh! Yeah that makes sense, thank you!